sandbox/experimental/data-packages/dwca_v2-dp/extended-occurrence-dp/0.1/table-schemas/survey.json

```diff
@@ -1,7 +1,8 @@
 {
-    "identifier": "http://rs.gbif.org/data-packages/dwca_v2/survey",
-    "url": "http://rs.gbif.org/sandbox/data-packages/dwca_v2/0.1/table-schemas/extended-occurrence-dp/survey.json",
     "name": "survey",
+    "identifier": "http://rs.gbif.org/data-packages/dwca_v2/extended-occurrence-dp/survey",
+    "url": "http://rs.gbif.org/sandbox/data-packages/dwca_v2/0.1/table-schemas/extended-occurrence-dp/survey.json",
```

Comment:
I do not think it should be dwca_v2-dp. This is an outstanding discussion with @timrobertson100. To me dwca_v2 has to be an equivalent thing to a dwca_v1. A dwca-v1 is a Darwin Core Archive - a specification for how make profiles of how files can be connected to each other with a core and one-level-deep extensions. To me, something called dwca_v2 has to be a specification for how to make profiles of how files can be connected to each other with required and optional tables at an arbitrary depth of connections. What we have is pseudo-frictionless data package that is one example of a dwca_v2 profile. I am very wary about confusing these two levels of things.

I chose extended_occurrence-dp because it suggests the family of data sets (Observations, MaterialGatherings, Surveys, OrganismInteractions and variations on these that support Occurrences) that can be publishing this way (an equivalent statement to an Event Core in dwca_v1). I'm not tied to the name of the data package, "dwca_v2_event-dp" would even be fine with me, but it must be a data package that implements dwca_v2, not dwca_v2 itself.

I hope that makes sense even if it is not what you were hoping for.

Reply:
I mean the URL is wrong, it won't resolve. I suppose it should be
`http://rs.gbif.org/sandbox/experimental/data-packages/dwca_v2-dp/extended-occurrence-dp/0.1/table-schemas/survey.json`